Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN apt-get update && \
         python3-setuptools python3-wheel python3-yaml python3-distro python3-jsonschema python3-newt \
         gosu lsb-release file vim less procps tree tar bzip2 zstd pigz bc tmux libncurses-dev \
         dosfstools mtools parted lz4 \
+        libgnutls28-dev \
```

Comment:
Please point to the Yocto version where this became a pre-installation dependency. I think I recently scanned them and didn't spot this. Maybe a downstream layer issue?

Reply:
I built Yocto for Variscite iMX board with new linux kernel (5.15.x).

Preinstalled dependencies were described [here](https://variwiki.com/index.php?title=Yocto_Build_Release&release=mx8mp-yocto-kirkstone-5.15-2.0.x-v1.0) for Variscite Yocto SDK.

You should read section 1 fro provided link for accessing required dependencies.

Reply:
Here is the official dependency list of kirkstone: https://docs.yoctoproject.org/4.0.3/brief-yoctoprojectqs/index.html#build-host-packages. We cannot address special cases of downstream layers, even less as there is a mechanism in Yocto to add missing dependencies via self-built packages ("gnutls-native").

I would recommend reporting to your supplier this problem so that they can fix it in their layer. Meanwhile, a bbappend may do the trick.

Reply:
Thank you for describing. 

I'm closing the MR.